crates/swc_ecma_transforms_optimization/src/json_parse.rs

```diff
@@ -129,6 +129,13 @@ fn jsonify(e: Expr) -> Value {
         Expr::Lit(Lit::Num(Number { value, .. })) => Value::Number((value as i64).into()),
         Expr::Lit(Lit::Null(..)) => Value::Null,
         Expr::Lit(Lit::Bool(v)) => Value::Bool(v.value),
+        Expr::Tpl(Tpl { quasis, .. }) => Value::String(match quasis.get(0) {
```

Comment:
This will error for template literal with interpolation, because there's no check for length of `quasis`.

Reply:
we checked length of tpl.expr at here https://github.com/swc-project/swc/blob/7ec6e94cb05929c36e28a59670b23f54472079f9/crates/swc_ecma_utils/src/lib.rs#L1466. 
so I think at here the length of quasis will always be 0 or 1.

Reply:
Oh, you are right. Thanks!